pkg/aws/ec2/api/eni_cleanup.go

```diff
@@ -102,6 +108,9 @@ func (e *ENICleaner) Start(ctx context.Context) error {
 // interval between cycle 1 and 2 and hence can be safely deleted. And we can also conclude that Interface 1 was
 // created but not attached at the the time when 1st cycle ran and hence it should not be deleted.
 func (e *ENICleaner) cleanUpAvailableENIs() {
+	vpccniAvailableENICnt.Set(0)
```

Comment:
I think for gauge type you don't need to reset the value. Every time when you report a value you can use set() instead of inc().